### DISCUSSION TITLE
Recover session from another profile

### DISCUSSION THREAD

**Discussion by RafaelLinux** [sienori/Tab-Session-Manager#876] [Q&A] [answered]
I did yesterday a reset to Firefox, thinking that only will clean Firefox registry and some other own data. However, Firefox reset cleaned all, including extensions and Tab-Session-Manager autobackup sessions. Luckily, Firefox did a copy of my profile pre-reseted.

So, to recover my previous sessions of TSM, I exited Firefox and copied older profile "sessionstore-backups" folder to my new Firefox profile. After launch Firefox and clicking on TSM icon to see autobackup session, there is no trace of my previous sessions.

What am I doing wrong and what I need to do to recover old profile session autosaved by TSM???

Thank you

**Comment by sienori** [comment, marked as answer]
How to migrate sessions to another profile:
https://github.com/sienori/Tab-Session-Manager/wiki/Q&A:-Sessions-data-storage-location;-How-to-migrate-sessions-data-to-another-profile
